### PR TITLE
Synchronize project status between tabs

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/7493
Also I found that deleted statuses were still showing up on the summary page.

## Testing
**URL to test:** 

https://7493-sync-status--atd-moped-main.netlify.app/moped/projects

**Steps to test:**

Add a status on the summary tab
Switch to comments tab and edit status (or delete status)
switch back to summary tab and see that status reflects current state

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [ ] ~Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#gid=776973707)~

### CODE REVIEW

File: moped-editor/src/queries/project.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ export const SUMMARY_QUERY = gql`
         location
       }
       moped_proj_notes(
-        where: { project_note_type: { _eq: 2 } }
+        where: { project_note_type: { _eq: 2 }, status_id: {_eq: 1} }
```
Comment (chiaberry) [inline moped-editor/src/queries/project.js:57]: only get the non-deleted notes

File: moped-editor/src/views/projects/projectView/ProjectComments.js [hunk 2]
```diff
@@ -74,14 +74,19 @@ const useStyles = makeStyles(theme => ({
...
- // The zeroth item in the list is intentionally blank; the notes are 1-indexed.
-const projectNoteTypes = ['', 'Internal Note', 'Status Update', 'Timeline Notes'];
+// Lookup array to convert project note types to a human readable interpretation
+// The zeroth item in the list is intentionally blank; the notes are 1-indexed.
+const projectNoteTypes = [
+  "",
+  "Internal Note",
+  "Status Update",
+  "Timeline Notes",
+];
```
Comment (chiaberry) [inline moped-editor/src/views/projects/projectView/ProjectComments.js:87]: this is just prettier formatting

File: moped-editor/src/views/projects/projectView/ProjectComments.js [hunk 3]
```diff
@@ -271,8 +280,9 @@ const ProjectComments = () => {
                 <List className={classes.root}>
                   {data.moped_proj_notes.map((item, i) => {
                     let isNotLastItem = i < data.moped_proj_notes.length - 1;
-                    let editableComment = (userSessionData.user_id === item.added_by_user_id ||
-                                           userHighestRole === "moped-admin");
+                    let editableComment =
```
Comment (chiaberry) [inline moped-editor/src/views/projects/projectView/ProjectComments.js:283]: from here down is just prettier

File: moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummary.js [hunk 4]
```diff
@@ -83,7 +83,6 @@ const ProjectSummary = ({ loading, error, data, refetch }) => {
   const { projectId } = useParams();
   const classes = useStyles();
 
-  const [makeSureRefresh, setMakeSureRefresh] = useState(false);
```
Comment (chiaberry) [inline moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummary.js:86]: not being used anywhere

File: moped-editor/src/views/projects/projectView/ProjectComments.js (hunk 3 above)
Comment (sergiogcx) [reply moped-editor/src/views/projects/projectView/ProjectComments.js:283]: any chance we could have those two variables made const?

**Review by sergiogcx — APPROVED** [review]

**Review by frankhereford — APPROVED** [review]
Works great - thank you!

File: moped-editor/src/views/projects/projectView/ProjectComments.js (hunk 3 above)
Comment (chiaberry) [reply moped-editor/src/views/projects/projectView/ProjectComments.js:283]: sure, the only change was me running prettier on the file, ill change them to const.

**Review by mateoclarke — APPROVED** [review]
🚢 💨 ✨

**Review by amenity — APPROVED** [review]
Super sturdy! I tried all the bug-prompting tricks and it worked flawlessly. Thanks, @chiaberry .

**Pull request merged**